models/models.go

```diff
@@ -0,0 +1,41 @@
+package models
+
+import (
+	"errors"
+	"time"
+)
+
+type Book struct {
+	ID       string     `json:"id" bson:"_id"`
+	Title    string     `json:"title"`
```

Comment:
It's worth adding bson tags to all fields, as we would rather the fields stored as camel_case (same as the json tags)